gateway/internal/config/router.go

```diff
@@ -48,12 +48,18 @@ func GetRouter(params *APIParams) *gin.Engine {
 		return NewContext(ctx, context.Background(), params.Logger)
 	}
 
+	subscriptionController := controllers.NewSubscriptionController(params.DispatchService)
+
 	r.GET("/rate", func(ctx *gin.Context) {
 		controllers.GetExchangeRate(newContext(ctx), params.CurrencyService)
```

Comment:
So why did you create Class only for subscription controller, but this left as it is?

Reply:
Rewrited rate controller as class